.github/workflows/rpm-build-and-test.yml

```diff
@@ -60,6 +60,9 @@ jobs:
              echo "text<<EOF" >> $GITHUB_OUTPUT;
              git show -s --format=%B | escape_single_quotes >> $GITHUB_OUTPUT;
              echo "EOF" >> $GITHUB_OUTPUT;
+      - name: Identify Commit Pragmas
+        run: . ci/gha_functions.sh;
+             echo '${{steps.dequoted_commit_message.outputs.text }}' | get_commit_pragmas
```

Comment:
The output from this is
https://github.com/daos-stack/daos/actions/runs/7170843997/job/19524500461
```
SKIP_CHECKPATCH=true\ be\ considered\ a\ commit\ pragma.
SKIP_BUILD=true
SKIP_PR_COMMENTS=true
REQUIRED_GITHOOKS=true
```

But in the commit message
```
'Skip-checkpatch: true' should not be considered a commit pragma.
Neither should
Skip-checkpatch: true be considered a commit pragma.
```

So is this working as intended?

Reply:
Yes.  That is a case I had hoped to be able to capture on the outset, but pragmas like `Test-tag:` require that we allow spaces in the value, eliminating a (space after the value) heuristic from being used to try to identify pragmas in the description.

This is reflected in the unit testing expected output at https://github.com/daos-stack/daos/blob/bmurrell/fix-make_commit_pragmas/ci/gha_functions.sh#L278C58-L278C126.

The PR comment has been updated to reflect that and should be used as the landing comment.

Reply:
Thanks, makes sense